.husky/prepare-commit-msg

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+node scripts/prepare-commit-message.js
```

Comment:
Any particular reason the `.husky` files were added? They don't seem related to the rest of this PR.

Reply:
Looks like `husky` was updated at some point and the new version has a breaking change that wasn't caught during the build because it is related to when you make a commit.